### PR TITLE
Fixed display of the player's name.

### PR DESCRIPTION
**Pull request by turbocat2001**
- Fixed display of the player's name; 
- Removed redundant use of vectors.

### CODE REVIEW

File: src/GameMenu.h [hunk 1]
```diff
@@ -31,12 +31,12 @@ class GameMenu {
...
     bool field_111;
     int nameCursorPos;
-    std::vector<int8_t> strArr;
+    char* strArr;
 
 public:
     int xPos;
 
-    GameMenu(std::string var1, Micro* micro, GameMenu* var3, std::vector<int8_t> var4);
+    GameMenu(std::string var1, Micro* micro, GameMenu* var3, char* imputData);
```
Comment (AntonEvmenenko) [inline src/GameMenu.h:39]: Typo, it should be "inputData" instead of "imputData"

Comment (AntonEvmenenko) [reply src/GameMenu.h:39]: Also, does it make sense to make "inputData" nullptr by default?

Comment (turbocat2001) [reply src/GameMenu.h]: @AntonEvmenenko Fixed

Comment (turbocat2001) [reply src/GameMenu.h]: Fixed: uses default nullptr value for inputString

**Pull request merged**